falcon-hello-world/gunicorn_config.py

```diff
@@ -0,0 +1 @@
+bind = '0.0.0.0:7010'
```

Comment:
Are you able to pick up the bind ip and port from environment values?

`bind = "{0}:{1}".format(os.getenv('bind_ip'), os.getenv('bind_port')`

What this would allow is setting these things dynamically through Docker Compose or commandline arguments with Docker. Though I guess the port is already exposed in the Dockerfile .. ?